### PR TITLE
Shahzeb/asterisk mixin

### PR DESCRIPTION
**Pull request by mshahzeb**
Asterisk mixin for Asterisk integration https://github.com/grafana/cloud-onboarding/issues/1175

Mixin with:
- Dashboards (Logs & Metrics)
- Alerts (Cortex based)

### CODE REVIEW

**Comment by mshahzeb** [comment]
The mixin currently have JSON based dashboards.

It is planned to convert them to grafonnet based dashboards in next phase once integration is live.

File: asterisk-mixin/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Asterisk Mixin
+
+The Asterisk Mixin is a set of configurable, reusable, and extensible alerts and
+dashboards based on the metrics exported by the [prometheus exporter in asterisk](https://wiki.asterisk.org/wiki/display/AST/Asterisk+18+Configuration_res_prometheus). The mixin also creates a dashboard that
```
Comment (rgeyer) [inline asterisk-mixin/README.md:4]: Should Asterisk be always capitalized?

```suggestion
dashboards based on the metrics exported by the [prometheus exporter in Asterisk](https://wiki.asterisk.org/wiki/display/AST/Asterisk+18+Configuration_res_prometheus). The mixin also creates a dashboard that
```

File: asterisk-mixin/dashboards/asterisk-logs.json [hunk 2]
```diff
@@ -0,0 +1,936 @@
...
+        "content": "<img style=\"margin: 20px;\" src=\"https://storage.googleapis.com/grafanalabs-integration-logos/asterisk.png\">",
+        "mode": "html"
+      },
+      "pluginVersion": "8.4.0",
+      "type": "text"
+    },
+    {
+      "datasource": {
+        "type": "loki",
+        "uid": "grafanacloud-logs"
```
Comment (rgeyer) [inline asterisk-mixin/dashboards/asterisk-logs.json:233]: This repeats in this file, there are several panels which are hardcoded to use your loki datasource uid. You can just do a search and replace.

```suggestion
        "uid": "${loki_datasource}"
```

**Review by rgeyer — CHANGES_REQUESTED** [review]
One small oversight, but otherwise it looks great!

File: asterisk-mixin/README.md (hunk 1 above)
Comment (mshahzeb) [reply asterisk-mixin/README.md]: Done - this is updated in latest commit

**Review by rgeyer — APPROVED** [review]
LGTM, thanks for the fixes. 🚀

**Pull request merged**